mosquitto/hooks/run

```diff
@@ -0,0 +1,5 @@
+#!{{pkgPathFor "core/bash"}}/bin/bash -xe
```

Comment:
Do we need bash for this or can we trim the package down by using `#!/bin/sh`

Reply:
Honestly, I don't know. I think not :)